README.md

```diff
@@ -24,15 +24,16 @@ Let this module handle file creation and systemd reloading.
 Or handle file creation yourself and trigger systemd.
 
 ```puppet
-include ::systemd
+include ::systemd::systemctl::daemon_reload
```

Comment:
I think I'd rather keep the interface in the `systemd` class and contain the subclasses (`systemd::systemctl::daemon_reload` and `systemd::tmpfiles`) for dependencies instead. Is there something that prevents from doing that?